ui/url_utils.py

```diff
@@ -0,0 +1,8 @@
+"""
+Utils for URLs (to avoid circular imports)
+"""
+
+DASHBOARD_URL = '/dashboard/'
+PROFILE_URL = '/profile/'
+PROFILE_PERSONAL_URL = '/profile/personal/'
```

Comment:
Would you mind getting rid of this one? Since our regex doesn't have a terminating character `/profile/` should match just as well